.github/workflows/docs_check.yml

```
@@ -13,10 +13,12 @@ on:  # yamllint disable-line rule:truthy
     branches:
       - master
     paths:
+      - 'CHANGELOG.md'
+      - 'README.md'
```

```suggestion
      - 'README.md'
      - 'SECURITY.md'
```

Security file is kinda the same type

---

True